docs/adr/adr-002-predevnet-core-to-full-communication.md

```diff
@@ -0,0 +1,49 @@
+# ADR for Devnet Celestia Core <> Celestia Node Communication 
+
+## Authors
+
+@renaynay @wondertan
```

Comment:
```suggestion
@renaynay @Wondertan
```

Reply:
I saw this after the second review, so this is why they are not aggregated into one